lib/CHeroHandler.cpp

```diff
@@ -277,11 +295,6 @@ CHeroHandler::CHeroHandler()
 {
 	VLC->heroh = this;
 
-	for (int i = 0; i < GameConstants::SKILL_QUANTITY; ++i)
-	{
-		VLC->modh->identifiers.registerObject("core", "skill", NSecondarySkill::names[i], i);
-		VLC->modh->identifiers.registerObject("core", "secondarySkill", NSecondarySkill::names[i], i);

```

Comment:
`secondarySkill` registration is missing now?

Reply:
fixed.